src/other/AutoCompleteText.js

```diff
@@ -71,8 +71,23 @@
                     var re = new RegExp("(" + search.split(' ').join("|") + ")", "gi");
                     return '<div class="autocomplete-suggestion" data-val="' + item.value + '" data-row="' + item.origRow + '">' + item.value.replace(re, "<b>$1</b>") + '</div>';
                 },
-                onSelect: function (e, term, item) {
-                    context.click(item.getAttribute("data-row"), context.valueColumn(), true);
```

Comment:
@GordonSmith , when rowToObj is added the result is dividing and placing each letter in the field 
Click:  {"Subject":"G","Year 1":"e","Year 2":"o","Year 3":"g","Year 4":"r"}, Subject, true.